app/assets/stylesheets/components/availability.scss

```
@@ -416,6 +416,7 @@
 }
 
 .availability--other-versions {
+  border-radius: var(--font-size-x-small);
```

What would you think about using
```suggestion
  border-radius: var(--border-radius-pill);
```

instead?  It's slightly more rounded (16px vs 12px), but I think it would be nice to not tie this box's border-radius to the font size (I think we'd find it unexpected in the future if we were to change the font size, and it has an effect on the box corner rounding).

---

Yes that sounds like a good change. I was set on 12px for some reason because I think one of the other rounded boxes had that border-radius. I was definitely iffy on using a font size variable for the border-radius but I didn't know where to find more appropriately-suited variables. Thanks for the fix!